### PR TITLE
[DUOS-2455][risk=no] Enable calls to add/remove data submitter role

### PR DESCRIPTION
**Pull request by rushtong**
## Addresses
https://broadworkbench.atlassian.net/browse/DUOS-2455

* Enables the issue/remove links for data submitters on the SO Console Data Submitters page
* Moves SO pages around so they are centrally located and more easily found.

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by fboulnois — APPROVED** [review]
👍 , I think at some point we should switch away from relative paths, but that is completely separate PR.

File: src/pages/signing_official_console/DataCustodianTable.js [hunk 1]
```diff
@@ -90,12 +92,12 @@ const IssueDataCustodianButton = (props) => {
...
 const researcherFilterFunction = getSearchFilterFunctions().signingOfficialResearchers;
 
-const CustodianCell = ({
+const SubmitterCell = ({
   researcher,
   showConfirmationModal
 }) => {
   const id = researcher.userId || researcher.email;
-  const button = researcher.isCustodian
+  const button = researcher.isDataSubmitter
```
Comment (connorlbark) [inline src/pages/signing_official_console/DataCustodianTable.js]: nit: would it be better to use a function, `isDataSubmitter(user)` here? I found the `setDataSubmitterProp` a bit unintuitive at first until I fully understood how it worked/interplayed with the rest of the system. i could also see someone in the future forgetting to call it, and not understanding why this button isn't showing up. totally understand if you'd rather just get it through, though.

**Review by connorlbark — APPROVED** [review]
This part looks good - although, when we issue the data submitter role, the text on the right column does not update with the added `DataSubmitter` until there is a refresh. Maybe this should be a separate ticket?

File: src/pages/signing_official_console/DataCustodianTable.js (hunk 1 above)
Comment (rushtong) [reply src/pages/signing_official_console/DataCustodianTable.js]: Yeah ... not proud of this bit of code. I was shooting for a quicker fix, but this is definitely a code smell. Will refactor that to be more consistent with how we're looking at other user roles.

**Review by fboulnois — APPROVED** [review]
🎉

**Pull request merged**